styles/about.css

```diff
@@ -0,0 +1,235 @@
+#about-hero {
+  background-size: cover;
+  background-image: url("../images/mobile-intro-bg.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  height: 35rem;
+  padding: 6rem 1.5rem;
+}
+
+.about-heading {
+  color: var(--orange);
+  text-align: center;
+}
+
+.about-heading h2 {
+  font-size: 1.7rem;
+  text-transform: uppercase;
+  font-weight: 900;
+  margin-top: 0.7rem;
+  margin-bottom: 2rem;
+}
+
+.about-heading p {
+  font-size: 1.4rem;
+  font-weight: 300;
+  margin-top: 0.2rem;
+}
+
+.about-desc {
+  border: 1px solid var(--lightGray);
+  padding: 1rem;
+  background-color: var(--white);
+  margin-bottom: 2rem;
+}
+
+.about-desc p {
+  color: var(--gray);
+  font-size: 0.9rem;
+  line-height: 1.3rem;
+  text-align: center;
+}
+
+.about-small {
+  color: var(--gray);
+  text-align: center;
+}
+
+.about-small strong {
+  color: var(--lightBlack);
+  text-decoration: underline;
+}
+
+#about-logo {
+  padding: 6rem 1.5rem;
+}
+
+.about-logo-heading .trait {
+  width: 1.5rem;
+  height: 0.2rem;
+  border-bottom: 1px solid var(--orange);
+  margin: 0 auto;
+}
+
+.about-logo-heading h1 {
+  text-align: center;
+  color: var(--lightBlack);
+  font-size: 1.2rem;
+  font-weight: 400;
+  margin-bottom: 0.7rem;
+}
+
+.about-logo-heading p {
+  font-size: 0.8rem;
+  color: var(--gray);
+  text-align: center;
+  margin-top: 1rem;
+}
+
+.about-logo-img {
+  height: 100px;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  border: 1px solid var(--lightGray);
+  margin-top: 2rem;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this, you could add the flex class to your `global.css`! 🤓